style.css

```diff
@@ -67,3 +69,284 @@ main {
 .icon li a {
   text-decoration: none;
 }
+
+.action img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: auto;
+  height: auto;
+}
+
+.works-title h1 {
+  text-align: center;
+}
+
+.works-title img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: auto;
+  height: auto;
+}
+
+.works-section {
+  display: grid;
+  justify-content: center;
+}
+
+.recent-works {
+  margin-left: auto;
+  margin-right: auto;
+  display: grid;
+  grid-template-rows: auto;
+  align-items: end;
+  margin-top: 75px;
+  background-color: #ebf0ee;
+  width: 327px;
+  height: 474px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.works-des {
+  text-align: center;
+  background-color: #fff;
+  border: 1px solid #d0d9d4;
+  color: #3a4a42;
+  height: 222px;
+  width: auto;
+  border-radius: 8px;
+  padding: 16px;
+}
+
+.works-des a {
+  text-decoration: none;
+  background-color: #ebf0ee;
+  color: #3a4a42;
+  border-radius: 4px;
+  padding: 8px 10px 8px 10px;
+}
+
+.works-des button {
+  background-color: #36b37e;
+  color: #fff;
+  margin-top: 26px;
+  border: none;
+  border-radius: 4px;
+  padding: 12px;
+}
+
+.works-des button a {
+  background-color: #36b37e;
+  color: #fff;
+  font-size: 17px;
+}
+
+.about-me {
+  background-image: url(./images/illustration\ 1-\ about\ me-mobile.png);
+  background-repeat: no-repeat;
+  background-position: right;
+  color: #172b4d;
+}
+
+.about-paragraph {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+}
+
+.resume {
+  background-image: url(./images/about-me-mobile-2.png);
+  background-repeat: no-repeat;
+  min-height: 58px;
+}
+
+.resume button {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 8px;
+  border-radius: 4px;
+  padding: 12px;
+  background-color: #36b37f;
+  border: none;
+  width: 159px;
+  height: 48px;
+}
+
+.resume button a {
+  text-decoration: none;
+  color: #fff;
+  font-size: 17px;
+}
+
+.language {
+  margin-left: auto;
+  margin-right: auto;
+  display: grid;
+  grid-template-rows: auto;
+  margin-top: 75px;
+  background-color: #ebf0ee;
+  width: 327px;
+  height: 352px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.language img {
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 40px;
+}
+
+.languages {
+  display: flex;
+  justify-content: center;
+}
+
+.languages a {
+  text-decoration: none;
+  margin: 0 10px 50px 10px;
+  background-color: #fff;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #36b37e;
+  border-radius: 8px;
+  padding: 12px;
+}
+
+.languages-h1 {
+  margin-top: -20px;
+}
```

Comment:
Kindly check and ensure the section has the all dimensions (font-size, line-height, font-weight etc) defined in the design and conforms so as to achieve a more flexible and uniform design in conformity with the original figma design. :+1:

Reply:
Fixed